test/write.cpp

```diff
@@ -54,27 +60,27 @@ template <typename T> auto sync_write(T const &t) {
 }
 } // namespace
 
-TEST_CASE("write a register", "[write]") {
-    using namespace groov::literals;
-    sync_write(grp("reg0"_r = 0xa5a5'a5a5u));
-    CHECK(data0 == 0xa5a5'a5a5u);
-}
-
-TEST_CASE("write a field", "[write]") {
-    using namespace groov::literals;
-    data0 = 0b1'1010'1u;
-    sync_write(grp("reg0.field1"_f = 1));
-    CHECK(data0 == 0b1'0001'1u);
-}
-
-TEST_CASE("write multiple fields to the same register", "[write]") {
-    using namespace groov::literals;
-    data0 = 0b1'1010'0u;
-    bus::num_writes = 0;
-    sync_write(grp("reg0.field0"_f = 1, "reg0.field1"_f = 0b101u));
-    CHECK(bus::num_writes == 1);
-    CHECK(data0 == 0b1'0101'1u);
-}
+// TEST_CASE("write a register", "[write]") {
```

Comment:
issue: commented tests should be removed or fixed

Reply:
oops :)